Categories/Adjoint/Instance/ZeroTruncation.agda

```diff
@@ -0,0 +1,99 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Adjoint.Instance.ZeroTruncation where
+
+-- The adjunction between 0-trucation and the inclusion functor from
+-- Setoids to Categories/Groupoids.
+
+open import Level using (Lift; lift)
+open import Data.Unit using (⊤; tt)
+import Function
+open import Relation.Binary using (Setoid)
+open import Function.Equality using (Π) renaming (id to idΠ)
+
+open import Categories.Adjoint
+open import Categories.Category.Instance.Groupoids
+open import Categories.Category.Instance.Setoids
+open import Categories.Functor renaming (id to idF)
+open import Categories.Functor.Instance.ZeroTruncation
+open import Categories.NaturalTransformation
+open import Categories.NaturalTransformation.NaturalIsomorphism
+
+-- The inclusion functor from Setoids to Groupoids
+
+Inclusion : ∀ {c ℓ} e → Functor (Setoids c ℓ) (Groupoids c ℓ e)
+Inclusion {c} {ℓ} e = record
+  { F₀ = Thin
+  ; F₁ = λ f → record
+    { F₀ = _⟨$⟩_ f
+    ; F₁ = cong f
+    ; identity     = lift tt
+    ; homomorphism = lift tt
+    ; F-resp-≈     = λ _ → lift tt
+    }
+  ; identity     = refl
+  ; homomorphism = refl
+  ; F-resp-≈     = λ {S T f g} f≗g →
+    let module S = Setoid S
+        module T = Setoid T
+    in record
+    { F⇒G = record { η = λ _ → f≗g S.refl         ; commute = λ _ → lift tt }
+    ; F⇐G = record { η = λ _ → T.sym (f≗g S.refl) ; commute = λ _ → lift tt }
+    ; iso = λ _ → record { isoˡ = lift tt ; isoʳ = lift tt }
+    }
+  }
+  where
+    open Π
+
+    Thin : Setoid c ℓ → Groupoid c ℓ e
```

Comment:
I think this function is useful enough to be exposed. In fact, I almost just re-implemented it!